kinetic/distribution.yaml

```diff
@@ -15723,6 +15723,16 @@ repositories:
       url: https://github.com/Intermodalics/tango_ros.git
       version: master
     status: developed
+  tb3_tools:
+    doc:
+      type: git
+      url: https://github.com/xuyuan-ict/tb3_tools.git
+      version: main
+    source:
+      type: git
+      url: https://github.com/xuyuan-ict/tb3_tools.git
```

Comment:
I've noticed a couple of problems in your repository:

1.  The license is declared in the package.xml (which is good), but there is no LICENSE file in the repository.  Can you please add one?
1.  You have checked in the `.pyc` files, which is not a good idea.  That will only work for the specific version of python that you used.  I'll suggest removing those files from the repository.
1.  The `tb3_safe_teleop` package depends on `message_generation`, but doesn't actually generate any messages.  I'll suggest you remove that dependency.

Reply:
Thank you for your valuable advice.
I have fixed these three problems in my repository : )